bacon.toml

```diff
@@ -0,0 +1,5 @@
+[jobs.fmt-check]
```

Comment:
Introducing this file is cool (I like having the formatting check as an option), but it breaks all the other standard `bacon` targets like `test` and `check-all`. Do we need to list them all to have them available?

I definitely wouldn't to merge this PR if it breaks how I typically use `bacon`. Should this be in a separate PR anyway?

Reply:
oh yea I could move it into it's own pr, was just there cuz I was using it myself and I forgot to exclude it from the commit. Afaik if there is no bacon config bacon just uses the default one, I don't know if there is some way to fallback to that, so we could either just copy the contents or I'll look into that.